src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -500,8 +519,8 @@ public static void GetFullPath_Windows_URIFormatNotSupported()
         [InlineData(@"C  :\somedir")]
         public static void GetFullPath_Windows_NotSupportedExceptionPaths(string path)
         {
-            // Many of these used to throw ArgumentException despite being documented as NotSupportedException
-            Assert.Throws<NotSupportedException>(() => Path.GetFullPath(path));
+            // Full Framework TFM < 4.6.2 throws ArgumentException. .Net Core and Full Framework TFM >= 4.6.2 throws NotSupportedException
+            AssertExtensions.ThrowsAny<ArgumentException, NotSupportedException>(() => Path.GetFullPath(path));
```

Comment:
On .NET Core I would like to explicitly check the exception type, so we don't accidentally change .NET Core behavior even if it's back to AE.